Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM library/node:latest
+FROM node:latest
 
-RUN mkdir /src
 WORKDIR /src
 
-COPY package.json .
-RUN npm install
+COPY package.json package-lock.json ./ 
+RUN npm ci
 
 COPY . .
-RUN tools/prepare.sh
+RUN npm build
```

Comment:
`npm build` is an error:
```
Step 6/9 : RUN npm build
 ---> Running in 2d57f00f9ad7
Top hits for "build"
————————————————————————————————————————————————————————————————————————————————
npm help install                                                         build:6
npm help config                                                          build:5
npm help explore                                                         build:5
npm help package-json                                                    build:4
npm help registry                                                        build:4
npm help rebuild                                                         build:4
npm help version                                                         build:4
npm help developers                                                      build:2
npm help semver                                                          build:2
npm help ci                                                              build:1
————————————————————————————————————————————————————————————————————————————————
(run with -l or --long to see more context)
```
It would be `npm run build`.

Weird thing that that be build succeeds (`docker build` succeeds) even though this RUN instruction fails.

Reply:
Weird indeed, and I always miss this. Thanks for the catch @pdcastro